README.md

```diff
@@ -290,8 +290,8 @@ function* naturals() {
   }
 }
 
-naturals().take(4).every(v => v > 1); // false, first value is 1
-naturals().take(4).every(v => v >= 1); // true, acting on a new iterator
+naturals().take(4).every(v => v > 0); // false, first value is 0
+naturals().take(4).every(v => v >= 0); // true, acting on a new iterator
```

Comment:
This example isn't great because it would also be true if it were _not_ acting on a new iterator, but whatever.